elastic/kibana
# [ILM] Hide rollover controls in advanced

## Summary

Improve the ILM form by moving rollover to an advanced section in the hot phase. Also refactor the description fields to have their enable toggles on the right under their descriptions.

Note all copy is temporary and still needs to be reviewed!

### Gif

![rollover-hidden](https://user-images.githubusercontent.com/8155004/100090979-4eda6c00-2e54-11eb-8d80-1b80376bb0fb.gif)


### Screenshots

moved form field toggles to the right under their descriptions:

<img width="1078" alt="Screenshot 2020-11-24 at 12 53 46" src="https://user-images.githubusercontent.com/8155004/100091034-60bc0f00-2e54-11eb-8328-70d059763c5a.png">

### How to test

This change is (mostly) aesthetic. ILM policy creation should work exactly as before. To test, create a new policy editing all fields and check the JSON in the show request flyout.

Also, load an existing policy in the UI and check that everything is as expected.

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
